docs/Reference/Pantheon-API-Methods.md

```diff
@@ -801,15 +801,15 @@ None
 
 ### eth_blockNumber
 
-Returns the index of the current block the client is processing.
+Returns the index of the block that the client is currently processing.
```

Comment:
I know this wasn't part of the original issue but while we're updating this I think we could use a more specific term than _processing_.  @ajsutton -> Can _processing_ be replaced with _importing_ here?

Reply:
I’d say this while description is wrong. It’s actually reporting the number of the current chain head (is the number of the block with highest total difficulty). The block we’re currently processing/importing is likely the next block after that (or it could be building on a different fork).

Reply:
Looking at other clients' definitions: 

[Parity's](https://wiki.parity.io/JSONRPC-eth-module.html#eth_blocknumber): `integer of the current block number the client is on.`

[Geth's](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_blocknumber): `integer of the current block number the client is on.`

Should we follow them, for a more universal definition?

Reply:
My preference would be 'Number of the current chain head' -> chain head is more specific than _on_

Reply:
would `integer corresponding to the current block number of the current chain head` be an appropriate description then ?

Reply:
`integer corresponding to the block number of the current chain head` would work.  The additional `current` suggests a block's number can change which it can't.